app/js/lib/foxbox/foxbox.js

```diff
@@ -38,103 +28,20 @@ let themes = [
   }
 ];
 
-const _private = {
+// Private members.
+const p = {
```

Comment:
I thought it was too verbose, so I shorten it.
But I really like this pattern and I'm reusing it in other parts of the lib.